api_mt/index.md

```diff
@@ -0,0 +1,137 @@
+
+  - [3.4.1 How to obtain an API key](#341-how-to-obtain-an-api-key)
+
+  - [3.4.2 How to set up the Memoq Translator Pro 9.8™ plugin for Google Cloud Translation Basic (or Google Translate API v2)](#342-how-to-set-up-the-memoq-translator-pro-98-plugin-for-google-cloud-translation-basic-or-google-translate-api-v2)
+
+[4. How to obtain API keys for MT plugins](#4-how-to-obtain-api-keys-for-mt-plugins)  
+
+* [4.1 Amazon Translate (AWS)](#41-amazon-translate-aws)
+
+* [4.2 DeepL](#42-deepl)
+
+* [4.3 ModernMT](#43-modernmt)
+
+* [4.4 Google Cloud Translation Basic (or Google Translate API v2)](#44-google-cloud-translation-basic-or-google-translate-api-v2)
+
+[5. References](#5-references)
+## 1. INTRODUCTION
+Machine Translation (MT) technology allows you to automatically translate your text from one language to another. Both Trados Studio 2021™ and memoQ translator pro 9.8™ use plugins to connect to machine translation engine vendors. This manual provides information as to how to obtain API keys from MT vendors and enter them into the plugins. 
+
+*Disclaimer*  
+
+Information contained herein is valid as of October 1st, 2021. Due to frequent changes in user interfaces utilized by MT vendors this information may become obsolete after that date without advance notice. Revision 1.0.
+### 1.1 MT vendors described
+This manual shows how to obtain APIs to the following MTs:
+
+| MT name | Advantages | Disadvantages | Cost |
+| ----------- | ----------- | ----------- | ----------- |
+| Google Cloud Translation Basic (or Google Translate API v2)  | best-known vendor, numerous language pairs available  | quality differs per language pair, very complicated set-up  | paid service with a free quota |
+| DeepL  |  very high quality, easy set-up  | limited number of language pairs  | paid service with a free trial |
+| Amazon Translate (AWS)   |  numerous language pairs available | quality differs per language pair, complicated set-up  | paid service with a free quota |
+| Modern MT   |  high-quality vendor, easy set-up | limited number of language pairs  | paid service with adaptive functions in a basic plan |
+
+### 1.2 Privacy
+More information on how the MT vendors listed in [Point 1.1](#11-mt-vendors-described) of this manual process your data is available from:
+
+ - for DeepL:
+ [https://www.deepl.com/pro-data-security/](https://www.deepl.com/pro-data-security/)
+ - for Google Cloud Translation Basic (or Google Translate API v2):
+ [https://cloud.google.com/translate/data-usage](https://cloud.google.com/translate/data-usage)
+ - Amazon Translate (AWS):
+ [https://docs.aws.amazon.com/translate/latest/dg/data-protection.html](https://docs.aws.amazon.com/translate/latest/dg/data-protection.html)
+ - Modern MT:
+ [https://www.modernmt.com/privacy/](https://www.modernmt.com/privacy/)
+
+[2. How to set up Trados Studio 2021™ plugins](Trados.md)
+
+[3. How to set up Memoq Translator Pro 9.8™ plugins](memoQ.md)
```

Comment:
They use memoQ, we also do elsewhere in this PR.

Reply:
Ok, thank you for pointing it out to me. I shall change it to ensure compatibility.

Reply:
Ok. The spelling is: memoQ now. Thank you.